### ISSUE TITLE
bug: Code autofill not working on inleed.net

### ISSUE THREAD

**Issue by Abragus** [twofas/2fas-browser-extension#84]
### Browser info

Firefox (Linux) version 126.0

### Browser extension version

1.7.0

### Website

https://login.inleed.net/2fa

### Bug description

When trying to autofill the 2FA code on Inleed's login page using the 2FAS extension, it "gets stuck" on the second input box, and tries to input the final four numbers there as well. The extension seems to correctly identify all input boxes, judging by the `data-twofas-element-number` added to the html elements, but numbers are still input incorrectly. 

Pasting all six numbers at once from the clipboard does work. 

I've included a video of the issue: 

[Inleed 2FAS autofill F12.webm](https://github.com/twofas/2fas-browser-extension/assets/43346912/dd135e64-d90d-4e4f-b4fc-e1bbb775433a)

### Actual HTML
```
<div class="d-flex flex-wrap flex-stack two-factor-inputs">
    <input wire:model.defer="input.0" type="number" class="two-step-code form-control form-control-solid h-sm-60px w-sm-60px w-50px h-50px fs-2qx text-center border-primary border-hover mx-sm-1 my-2  first-input ">
    <input wire:model.defer="input.1" type="number" class="two-step-code form-control form-control-solid h-sm-60px w-sm-60px w-50px h-50px fs-2qx text-center border-primary border-hover mx-sm-1 my-2 ">
    <input wire:model.defer="input.2" type="number" class="two-step-code form-control form-control-solid h-sm-60px w-sm-60px w-50px h-50px fs-2qx text-center border-primary border-hover mx-sm-1 my-2 ">
    <input wire:model.defer="input.3" type="number" class="two-step-code form-control form-control-solid h-sm-60px w-sm-60px w-50px h-50px fs-2qx text-center border-primary border-hover mx-sm-1 my-2 ">
    <input wire:model.defer="input.4" type="number" class="two-step-code form-control form-control-solid h-sm-60px w-sm-60px w-50px h-50px fs-2qx text-center border-primary border-hover mx-sm-1 my-2 ">
    <input wire:model.defer="input.5" type="number" class="two-step-code form-control form-control-solid h-sm-60px w-sm-60px w-50px h-50px fs-2qx text-center border-primary border-hover mx-sm-1 my-2  last-input ">
</div>
```

Please let me know if any other information is needed! 

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I understand that security vulnerabilities should be reported to security@2fas.com instead of on GitHub.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by GrzegorzZajac000** [comment]
@Abragus Thank you for your contribution. It will be fixed in v1.7.2. This version is now under our internal testing.

**Comment by GrzegorzZajac000** [comment]
@Abragus v1.7.2 was uploaded to stores. Should be available in next hours.
